### ISSUE TITLE
Improve issues management

### ISSUE THREAD

**Issue by mosabua** [trinodb/trino#23121]
The backlog of over 2.3k issues is unmaintainable. We need to clear this up and define a process for managing issues going forward.

Things to look at:

* use stalebot like for PRs
* do a large prune for existing issues but keep roadmap and other issues
* need to figure out specific policy and process for monitoring progress on open and new issue

**Comment by Akanksha-kedia** [comment]
@mosabua  We have enhance the template of issues, where sometimes issue is not finally an issues its suspected to be an issue so we can have some labels added while creating an issue so that its becomes easy to filter out.

**Comment by mosabua** [comment]
I saw some of these approaches already @Akanksha-kedia .. but you want to point out where you have done that?

**Comment by hashhar** [comment]
The idea of having a `triage` label applied automatically to all incoming issues is good.

It helps maintainers to see which issues need attention and then either closing it, moving discussion to Slack or confirming issue as a bug etc. can be done.
Right now I have no idea if an issue was already looked at (or being looked at) by someone else or not.

It would only help with preparing the queue though - what happens after things get triaged is the harder part IMO.

**Comment by mosabua** [comment]
yeah .. we want to do more than just that though .. still need to flesh it out .. automatic messaging, and closing after some time like the PRs and so on.

**Comment by lozbrown** [comment]
Suggest using the power of Gen AI you help you out here, at my employer we've used Gen AI to help triage our issue management, flagging issues as related or dupes. Ensuring issues have enough information (and requesting missing) and because the application team have been flagged as SMEs its been able to respond to issues raised with accurate responses on the causes of issues because its seen similar issues raised in the past and closed after comments by the app team. We've had great success where users have closed the issues because the bot has successfully addresses the issues with comments. 

a quick google find this 
https://dev.to/maurerkrisztian/building-an-innovative-ai-tool-to-revolutionize-github-issues-management-14bb


I get that the current situation is unmanageable, and getting out of that is a gargantuan task. But any cull of issues that are seen to be 'stale' risks leading to community feeling un-cared for and unaddressed. 

In reality all github issues are either, not an issue, responses needed to make the user see that. OR are an issue, probably needs to be address or at least marked as not a priority, 
Anything you can get to help with that would be an improvement.
